### PR TITLE
feat(trends): Fetch count timeseries and pass it to breakpoint microservice

### PR DESCRIPTION
**Pull request by udameli**
To display counts before and after breakpoint, microservice needs event counts for each bucket. This will cause errors in microservice since the request shape is changing cc: @npusarla

### CODE REVIEW

File: src/sentry/api/endpoints/organization_events_new_trends.py [hunk 1]
```diff
@@ -133,8 +136,8 @@ def get_event_stats(
             trends_request["data"] = response.data
 
             # get start and end from the first transaction
-            trends_request["start"] = response.data[list(response.data)[0]]["start"]
-            trends_request["end"] = response.data[list(response.data)[0]]["end"]
+            trends_request["start"] = response.data[list(response.data)[0]][trend_function]["start"]
```
[the remark this answers is not in the document]
Comment (udameli) [reply src/sentry/api/endpoints/organization_events_new_trends.py:139]: ^ @npusarla

[the remark this answers is not in the document]
Comment (npusarla) [reply src/sentry/api/endpoints/organization_events_new_trends.py:139]: sure - but wouldn't all the transactions being passed through to the microservice have the same time range anyways?

[the remark this answers is not in the document]
Comment (udameli) [reply src/sentry/api/endpoints/organization_events_new_trends.py:139]: for now! but it'd be nice to not have implicit assumptions that it's always going to be like that

[the remark this answers is not in the document]
Comment (udameli) [reply src/sentry/api/endpoints/organization_events_new_trends.py:139]: updated the spec to show where start and end are for each transaction
image

**Review by npusarla — APPROVED** [review]
LGTM

**Pull request merged**